### PR TITLE
Closes #1313: Adding IntelligenceBank Drupal Connector configuration

### LINKED ISSUE #1313: Explore options for adding the DAM Intelligence Bank Drupal connector to Quickstart 2.
## What is the problem that we want to solve?
The university has adopted Intelligence Bank as the Digital Asset Manager for the university to make it easier to share images, videos and other digital assets around the whole university. 

Intelligence Bank provides a couple of different options for granting access to this DAM within Drupal sites: https://www.drupal.org/project/intelligencebank

This spike is to determine what the best path forward is to incorporate access to Intelligence Bank inside of Quickstart 2

## Conditions of satisfaction
- [ ] The considered options are documented
- [ ] A recommended next step is documented
- [ ] An estimate of time and effort is provided for the recommendation

### PR DESCRIPTION
**Pull request by gflora**
## Related issues
#1313 

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
- Minor release changes
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- [ ] Other or unknown

### Drupal core
- Patch release changes
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major or minor level update
- [ ] Other or unknown

### Drupal contrib projects
- Patch release changes
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- Minor release changes
   - [ ] Major level update
- [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+uuid: 3e3c5ab9-8755-422a-a3db-661b855c0d2a
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml:1]: We omit `uuid`s from exported config to make the config reusable across multiple sites.
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+status: true
+dependencies:
+  config:
+    - field.field.media.ib_dam_embed.field_media_ib_dam_embed
+    - media.type.ib_dam_embed
+  module:
+    - link
+    - path
+_core:
+  default_config_hash: 5z7JVAF7hj_6rtGjz8u6N_YkYqgESCg0-JIL93lY1ok
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml:12]: We also omit these default config hash values from exported configs.
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_demo_page.default.yml [hunk 3]
```diff
@@ -0,0 +1,100 @@
+uuid: cc301f99-a449-482d-9482-79a993d473d8
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_demo_page.default.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_media.default.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
+uuid: 6be41c47-fe1b-4c11-88f2-76669d64953e
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_media.default.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
+uuid: b2f16ee4-7096-4d1c-a77e-43a1185a9475
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.media.ib_dam_embed.field_media_ib_dam_embed
+    - media.type.ib_dam_embed
+  module:
+    - ib_dam
+_core:
+  default_config_hash: rY3bCqMOFUDYqHzQJKkJQ23jkrFvgOAbuvtzkTzHha0
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml:11]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.default.yml [hunk 7]
```diff
@@ -0,0 +1,38 @@
+uuid: 6da96317-1d5f-4c66-ad69-71a16f079ef5
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.default.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.teaser.yml [hunk 8]
```diff
@@ -0,0 +1,32 @@
+uuid: 37e02b5e-33a3-48e6-986c-d798602aff9a
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.teaser.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_media.default.yml [hunk 9]
```diff
@@ -0,0 +1,29 @@
+uuid: ec9cb7b3-5791-4ebe-a525-46b9bc34447c
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_media.default.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_media.teaser.yml [hunk 10]
```diff
@@ -0,0 +1,22 @@
+uuid: 0cd3754d-0e05-4123-bd37-1e14a12e3f99
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_media.teaser.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml [hunk 11]
```diff
@@ -0,0 +1,110 @@
+uuid: 2fe82e3f-9580-4c7f-9883-1697338c17a9
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.az_standard
+  module:
+    - ckeditor
+_core:
+  default_config_hash: p5wkWxOoV1gCUiMWuf-xQvDMVX2eCZhaQ_6nQwLnPYE
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml:10]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml [hunk 13]
```diff
@@ -0,0 +1,85 @@
+uuid: 3edeafcb-89a2-4215-b036-7d2f30157afe
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.basic_html
+  module:
+    - ckeditor
+_core:
+  default_config_hash: AqlPmO16LvJI4D0Ih6u4GFQIzqr5OnLgAUSjcUGWk2g
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml:10]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/editor.editor.full_html.yml [hunk 15]
```diff
@@ -0,0 +1,79 @@
+uuid: 0d90b549-88cc-42c1-a7f2-f813604f384f
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/editor.editor.full_html.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/entity_browser.browser.ib_dam.yml [hunk 16]
```diff
@@ -0,0 +1,28 @@
+uuid: 86e23d29-bcfd-4d90-ac08-d9ba6c8d6957
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/entity_browser.browser.ib_dam.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml [hunk 17]
```diff
@@ -0,0 +1,25 @@
+uuid: 315a190d-a53f-4eb9-9e0f-9e3a95b8f13b
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_ib_dam_embed
+    - media.type.ib_dam_embed
+  module:
+    - link
+_core:
+  default_config_hash: p1KexYTRAx9iPcnr_0I3rToTWFuUinxf2M7EZeNAJig
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml:11]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.field.node.az_flexible_page.field_ib_testing.yml [hunk 19]
```diff
@@ -0,0 +1,39 @@
+uuid: bd1e7524-058a-44c1-9566-47f129b28d8a
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.field.node.az_flexible_page.field_ib_testing.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.field.node.ib_demo_page.field_ib_media.yml [hunk 20]
```diff
@@ -0,0 +1,39 @@
+uuid: 62fab722-44d9-4423-8f89-082eb9332290
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.field.node.ib_demo_page.field_ib_media.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.field.node.ib_media.field_ib_media.yml [hunk 21]
```diff
@@ -0,0 +1,39 @@
+uuid: a3647594-0a5a-4820-b84b-048f269f36c7
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.field.node.ib_media.field_ib_media.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml [hunk 22]
```diff
@@ -0,0 +1,21 @@
+uuid: 7c6526e4-ef8d-4040-adb3-967c1919ce9a
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml [hunk 23]
```diff
@@ -0,0 +1,21 @@
...
+langcode: en
+status: true
+dependencies:
+  module:
+    - link
+    - media
+_core:
+  default_config_hash: v0xW1nC9g8gauXZZwTrKrhmhNO4bzuwD5beQ-1udJJ4
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml:9]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_media.yml [hunk 24]
```diff
@@ -0,0 +1,20 @@
+uuid: 8829ee68-0e15-434f-abc6-2f40575a9fa3
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_media.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_testing.yml [hunk 25]
```diff
@@ -0,0 +1,20 @@
+uuid: 73824dea-ceae-4d05-9865-40416ecb0b20
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_testing.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml [hunk 26]
```diff
@@ -0,0 +1,135 @@
+uuid: dbbff4a1-e9dc-4942-b91c-6533223d9055
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml [hunk 27]
```diff
@@ -0,0 +1,135 @@
...
+    - core.entity_view_mode.media.az_small
+    - core.entity_view_mode.media.az_square
+  module:
+    - bootstrap_utilities
+    - editor
+    - entity_embed
+    - ib_dam_wysiwyg
+    - media
+_core:
+  default_config_hash: Pf57ySytPg-TGLPWRmY40p-5YBwtdV07mz5mnKZYCoo
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml:18]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml [hunk 28]
```diff
@@ -0,0 +1,74 @@
+uuid: d0c85e92-3483-4935-9664-ec2b98c1af5c
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml [hunk 29]
```diff
@@ -0,0 +1,74 @@
...
+langcode: en
+status: true
+dependencies:
+  module:
+    - bootstrap_utilities
+    - editor
+    - ib_dam_wysiwyg
+    - media
+_core:
+  default_config_hash: vxZvKWwZReEd1yDpL5Mmxyj5-8fRTjHAGqJaIaq7toA
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml:11]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/filter.format.full_html.yml [hunk 30]
```diff
@@ -0,0 +1,81 @@
+uuid: 85c5cb39-a1ef-4934-9d60-f9e18d0f1852
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/filter.format.full_html.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/filter.format.full_html.yml [hunk 31]
```diff
@@ -0,0 +1,81 @@
...
+status: true
+dependencies:
+  module:
+    - bootstrap_utilities
+    - editor
+    - entity_embed
+    - ib_dam_wysiwyg
+    - media
+_core:
+  default_config_hash: 9azY1oSM0Hjq8apA4EngWlcDkd1wqBG-6blbsB2fF9k
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/filter.format.full_html.yml:12]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml [hunk 32]
```diff
@@ -0,0 +1,19 @@
+uuid: 810cdd1a-2e58-4dca-8624-b7679c64604d
+langcode: en
+status: true
+dependencies:
+  module:
+    - ib_dam_media
+_core:
+  default_config_hash: fU_Gpc4sFjPv9BQ935SP8F77Nt0DqJRIbRYQed_Bbis
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml:8]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml [hunk 33]
```diff
@@ -0,0 +1,19 @@
+uuid: 810cdd1a-2e58-4dca-8624-b7679c64604d
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/node.type.ib_demo_page.yml [hunk 34]
```diff
@@ -0,0 +1,18 @@
+uuid: 64a83cac-4f3c-4eb5-aa4e-d0847332b015
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/node.type.ib_demo_page.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/node.type.ib_media.yml [hunk 35]
```diff
@@ -0,0 +1,18 @@
+uuid: 54899adf-0d2e-4225-92e6-77e52aecec40
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/node.type.ib_media.yml:1]:
```suggestion
```

File: modules/custom/az_dam/config/quickstart/config_snapshot.snapshot.config_sync.module.ib_dam_media.yml [hunk 36]
```diff
@@ -0,0 +1,193 @@
+uuid: 49a1b786-ff3b-44c9-bf22-898c4b95b0af
```
Comment (joeparsons) [inline modules/custom/az_dam/config/quickstart/config_snapshot.snapshot.config_sync.module.ib_dam_media.yml:1]: I don't think we want to include this exported config snapshot

**Review by joeparsons — CHANGES_REQUESTED** [review]
Thanks for your PR, @gflora!

I've added some commit suggestions for removing `uuid` and `_core: default_config_hash:` values from the exported config files.  We omit those to ensure the exported config is usable on any site.

I think we also probably need to move all of the files you have in `modules/custom/az_dam/config/quickstart` into a new folder called `modules/custom/az_dam/config/install` except for these:

- modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml
- modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml
- modules/custom/az_dam/config/quickstart/editor.editor.full_html.yml
- modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml
- modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml
- modules/custom/az_dam/config/quickstart/filter.format.full_html.yml
- modules/custom/az_dam/config/quickstart/ib_dam.settings.yml
- modules/custom/az_dam/config/quickstart/ib_dam_media.settings.yml

I also don't think we want to include the exported config_snapshot file.

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml (hunk 1 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml (hunk 2 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml:12]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_demo_page.default.yml (hunk 3 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_demo_page.default.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_media.default.yml (hunk 4 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_form_display.node.ib_media.default.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml (hunk 5 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml (hunk 6 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_view_display.media.ib_dam_embed.default.yml:11]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.default.yml (hunk 7 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.default.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.teaser.yml (hunk 8 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.teaser.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_media.default.yml (hunk 9 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_media.default.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml (hunk 11 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml (hunk 12 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/editor.editor.az_standard.yml:10]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml (hunk 13 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml (hunk 14 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/editor.editor.basic_html.yml:10]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/editor.editor.full_html.yml (hunk 15 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/editor.editor.full_html.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/entity_browser.browser.ib_dam.yml (hunk 16 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/entity_browser.browser.ib_dam.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml (hunk 17 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml (hunk 18 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.field.media.ib_dam_embed.field_media_ib_dam_embed.yml:11]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.field.node.az_flexible_page.field_ib_testing.yml (hunk 19 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.field.node.az_flexible_page.field_ib_testing.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.field.node.ib_demo_page.field_ib_media.yml (hunk 20 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.field.node.ib_demo_page.field_ib_media.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.field.node.ib_media.field_ib_media.yml (hunk 21 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.field.node.ib_media.field_ib_media.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml (hunk 22 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml (hunk 23 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.storage.media.field_media_ib_dam_embed.yml:9]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_media.yml (hunk 24 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_media.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_testing.yml (hunk 25 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/field.storage.node.field_ib_testing.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml (hunk 26 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml (hunk 27 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/filter.format.az_standard.yml:18]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml (hunk 28 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml (hunk 29 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/filter.format.basic_html.yml:11]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/filter.format.full_html.yml (hunk 30 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/filter.format.full_html.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/filter.format.full_html.yml (hunk 31 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/filter.format.full_html.yml:12]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml (hunk 32 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml:8]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml (hunk 33 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/node.type.ib_media.yml (hunk 35 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/node.type.ib_media.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/config/quickstart/node.type.ib_demo_page.yml (hunk 34 above)
Comment (gflora) [reply modules/custom/az_dam/config/quickstart/node.type.ib_demo_page.yml:1]: Resolved locally. Will commit for review when all changes have been made.

File: modules/custom/az_dam/az_dam.info.yml [hunk 37]
```diff
@@ -0,0 +1,6 @@
+name: 'Quickstart DAM'
+type: module
+core_version_requirement: ^9
+package: The University of Arizona
+dependencies:
+  - intelligencebank:intelligencebank
```
Comment (joeparsons) [inline modules/custom/az_dam/az_dam.info.yml:6]: I think this needs to be `ib_dam`.  `intelligencebank` is the package name and it contains submodules.
```suggestion
  - intelligencebank:ib_dam
```

Comment (joeparsons) [reply modules/custom/az_dam/az_dam.info.yml:6]: We may also want to add these as dependencies to `az_dam`:

```
  - intelligencebank:ib_dam_media
  - intelligencebank:ib_dam_wysiwyg
```

Comment (trackleft) [inline modules/custom/az_dam/az_dam.info.yml:6]:
```suggestion
  - ib_dam:ib_dam
  - ib_dam:ib_dam_media
  - ib_dam:ib_dam_wysiwyg

```

File: composer.json [hunk 38]
```diff
@@ -67,6 +67,7 @@
         "drupal/google_tag": "1.4",
         "drupal/honeypot": "2.0.2",
         "drupal/image_widget_crop": "2.3",
+        "drupal/intelligencebank":"2.4",
```
Comment (trackleft) [inline composer.json:70]:
```suggestion
        "drupal/intelligencebank":"2.4",   
        "drupal/entity_browser": "2.6",
```

File: modules/custom/az_dam/az_dam.info.yml (hunk 37 above)
Comment (joeparsons) [reply modules/custom/az_dam/az_dam.info.yml:6]: Shouldn't these be in this format?
```
  - intelligencebank:ib_dam
  - intelligencebank:ib_dam_media
  - intelligencebank:ib_dam_wysiwyg
```

Comment (joeparsons) [reply modules/custom/az_dam/az_dam.info.yml:6]: @trackleft Shouldn't these be in this format?
```
  - intelligencebank:ib_dam
  - intelligencebank:ib_dam_media
  - intelligencebank:ib_dam_wysiwyg
```

File: modules/custom/az_dam/az_dam.info.yml [hunk 39]
```diff
@@ -0,0 +1,6 @@
+name: 'Quickstart DAM'
+type: module
```
Comment (trackleft) [inline modules/custom/az_dam/az_dam.info.yml:2]:
```suggestion
type: module
description:  'IntelligenceBank Drupal connector configuration'
```

File: modules/custom/az_dam/az_dam.info.yml (hunk 37 above)
Comment (trackleft) [reply modules/custom/az_dam/az_dam.info.yml:7]: It worked the other way, but I'm really not sure, the composer.json file calls it drupal/ib_dam
https://git.drupalcode.org/project/intelligencebank/-/blob/8.x-2.x/composer.json#L2

**Comment by trackleft** [comment]
- [ ] Unable to install Quickstart DAM due to unmet dependencies: core.entity_form_display.node.ib_demo_page.default (field.field.node.ib_demo_page.body)

**Comment by camikazegreen** [comment]
Superseded by #1592

**Pull request closed without merging**